apps/frontend/src/libs/modules/notifier/notifier.module.ts

```diff
@@ -0,0 +1,21 @@
+import { toast } from "react-toastify";
+
+class ToastNotifier {
+	public showError(message?: string): void {
```

Comment:
message should be required
```suggestion
	public showError(message: string): void {
```